recipe/meta.yaml

```diff
@@ -49,7 +49,7 @@ requirements:
     #  against liblzo."
     # - lzo
     - openssl {{ openssl }} # [not osx]
-    - libxml2 2.10
+    - libxml2 {{ libxml2 }}
     - zlib {{ zlib }}
     - zstd {{ zstd }}
   run:
```

Comment:
`pin_compatible(`openssl`, x.y.z)` ? unless there's run_exports in openssl?

Reply:
I think a plain openssl is enough. I didn't find the usage pin_compatible for openssl. Other recipes have openssl in `run` if `openssl {{ openssl }}` in `host`, for example, https://github.com/AnacondaRecipes/arrow-cpp-feedstock/blob/master/recipe/meta.yaml#L83

Reply:
I replaced it with `openssl`

Reply:
yep, there's a run_exports in the openssl package that will take care of compatibility